webapp/tests/karma/ts/providers/translate-utils.provider.spec.ts

```diff
@@ -0,0 +1,143 @@
+import sinon from 'sinon';
+import { expect } from 'chai';
+import * as MessageFormat from 'messageformat';
+
+import {
+  TranslateMessageFormatCompilerProvider,
+  TranslateParserProvider,
+} from '@mm-providers/translate-utils.provider';
+
+let provider;
+describe('TranslateUtils providers', () => {
+  afterEach(() => {
+    sinon.restore();
+  });
+
+  describe('Translate MessageFormat compiler provider', () => {
```

Comment:
Maybe add some spaces between "test data", "unit to test" and "asserts" when there's 1 scenario in the "it", improves readability.